include/tvm/relay/type.h

```diff
@@ -82,6 +82,9 @@ class TensorTypeNode : public BaseTensorTypeNode {
     v->Visit("span", &span);
   }
 
+  /*! \brief Return product of elements in the shape */
```

Comment:
Could we add one more line to the docs such as `(d_1, d_2, ..., d_n) -> d1 * d_2 ... * d_n`.